src/reuse/modules/util/file.ts

```diff
@@ -44,7 +44,7 @@ export class File {
         const filePath = path.resolve(file);
         vl.log(`Uploading file with a path ${filePath}`);
         const remoteFilePath = await browser.uploadFile(filePath);
-        await elem.addValue(remoteFilePath);
+        await elem.setValue(remoteFilePath);
```

Comment:
@heraldantony mentioned, that setValue is only possible for displayed elements. Are we sure it is working with that change for hidden inputs?

Reply:
@DIFSRIP for hidden element explicitly they have handle it externally to make visible samething we followed in customerE2E.if we want me to handle it at util level then I will do it.

Reply:
are there many such cases where it is not visible, do we have concrete samples?

Reply:
We can improve it in a separate PR if so, need to release the fix soon, so I will merge it like this for now.